code/game/mob/living/carbon/human/unarmed_attack.dm

```diff
@@ -14,8 +14,8 @@ var/global/list/sparring_attack_cache = list()
 	var/deal_halloss
 	var/sparring_variant_type = /datum/unarmed_attack/light_strike
 
-	var/eye_attack_text
-	var/eye_attack_text_victim
+	var/eye_attack_text = "fingers"
```

Comment:
Why???

Reply:
Because the only instance we use this is for humans, it doesn't grab the variable from the datum due to how it works.